### PR TITLE
Pei-Ling 24

### PR DESCRIPTION
**Pull request by wuberde**

### CODE REVIEW

File: 24-PB-function-declaring/Pei-Ling.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+`use strict`;
+
+// #### 1. Multiply - Function Declaration
+function myMultiply1(a, b) {
+    return a * b;
+}
+console.log(`Task 1: Function "myMultiply1" definition = a * b`);
+
+// #### 2. Multiply - Function Declarations as Values
+function myMultiply2(a = 2, b = 3) {
```
Comment (martl) [inline 24-PB-function-declaring/Pei-Ling.js:10]: yeah here you should use the anonymous function: 
const myMultiply2 = function....

File: 24-PB-function-declaring/Pei-Ling.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+let myMultiply3 = (a, b) => {
+    return a * b;
+}
+console.log(`Task 3:`, myMultiply3(5, 4));
+
+// #### 4. Declarations
+const myMultiply4 = (a, b) => {
+    return a % b;
+}
+console.log(`Task 4:`, myMultiply4(9, 8));
```
Comment (martl) [inline 24-PB-function-declaring/Pei-Ling.js:25]: number 4 should also be written in all three ways, here you did only Arrow function way. So please add it as "normal function bla ()... " and as an anonymous function like i showed before.

File: 24-PB-function-declaring/Pei-Ling.js (hunk 1 above)
Comment (wuberde) [reply 24-PB-function-declaring/Pei-Ling.js:10]: updated

File: 24-PB-function-declaring/Pei-Ling.js (hunk 2 above)
Comment (wuberde) [reply 24-PB-function-declaring/Pei-Ling.js:25]: updated

**Pull request closed without merging**